pom.xml

```diff
@@ -123,28 +123,28 @@
             </dependency>
             <dependency>
                 <groupId>org.powermock</groupId>
-                <artifactId>powermock-api-mockito</artifactId>
+                <artifactId>powermock-api-mockito2</artifactId>
                 <version>${powermock.version}</version>
                 <scope>test</scope>
             </dependency>
         </dependencies>
     </dependencyManagement>
 
     <properties>
-        <carbon.kernel.version>4.7.0</carbon.kernel.version>
+        <carbon.kernel.version>4.7.1</carbon.kernel.version>
         <carbon.kernel.package.import.version.range>[4.4.0, 5.0.0)</carbon.kernel.package.import.version.range>
         <carbon.kernel.registry.imp.pkg.version>[1.0.1, 2.0.0)</carbon.kernel.registry.imp.pkg.version>
         <osgi.service.component.imp.pkg.version.range>[1.2.0, 2.0.0)</osgi.service.component.imp.pkg.version.range>
         <osgi.service.http.imp.pkg.version.range>[1.2.1, 2.0.0)</osgi.service.http.imp.pkg.version.range>
         <osgi.framework.imp.pkg.version.range>[1.7.0, 2.0.0)</osgi.framework.imp.pkg.version.range>
         <osgi.util.tracker.imp.pkg.version.range>[1.5.1, 2.0.0)</osgi.util.tracker.imp.pkg.version.range>
         <!--Carbon commons version-->
-        <carbon.commons.version>4.4.8</carbon.commons.version>
+        <carbon.commons.version>4.8.7</carbon.commons.version>
         <carbon.commons.imp.pkg.version>[4.4.0, 5.0.0)</carbon.commons.imp.pkg.version>
         <!--Carbon identity version-->
-        <carbon.identity.framework.version>5.20.34</carbon.identity.framework.version>
+        <carbon.identity.framework.version>5.23.28</carbon.identity.framework.version>
 
-        <carbon.identity.version>5.0.7</carbon.identity.version>
+        <carbon.identity.version>5.23.28</carbon.identity.version>
```

Comment:
why did you bump the carbon.identity version? Do we have this version

Reply:
Yes we have this version.
https://github.com/wso2/carbon-identity-framework/releases/tag/v5.23.28

Reply:
That is already there as carbon.identity.framework.version

Reply:
I will revert it to the 5.0.8, which is the latest version for the carbon.identity repo